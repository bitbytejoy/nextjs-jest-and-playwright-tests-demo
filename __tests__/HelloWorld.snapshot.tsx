import {render} from "@testing-library/react";
import HelloWorld from "../components/HelloWorld";

it('renders a hello world message', () => {
  const {container} = render(<HelloWorld />);
  expect(container).toMatchSnapshot();
});