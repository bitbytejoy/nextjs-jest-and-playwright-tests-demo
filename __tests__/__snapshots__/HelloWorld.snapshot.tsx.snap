// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders a hello world message 1`] = `
<div>
  <div>
    <h1>
      Hello World!
    </h1>
  </div>
</div>
`;
